static/src/javascripts/projects/common/modules/experiments/tests/contributions-membership-epic-backup.js

```diff
@@ -0,0 +1,123 @@
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic-equal-buttons.html',
+    'text!common/views/contributions-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/modules/experiments/embed',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features',
+    'lodash/arrays/intersection'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpicEqualButtons,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             embed,
+             ajax,
+             commercialFeatures,
+             intersection) {
+
+
+    return function () {
+
+        this.id = 'ContributionsMembershipEpicBackup';
+        this.start = '2016-11-08';
+        this.expiry = '2016-11-11';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Back up contributions-only epic in case membership goes down';
+        this.showForSensitive = false;
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Impressions to number of contributions/supporter signups';
+        this.audienceCriteria = 'All readers reading about US politics/election';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'If membership goes down, we have a backup way of asking for support from our readers';
+        this.canRun = function () {
+            var whitelistedKeywordIds = ['us-news/us-elections-2016', 'us-news/us-politics'];
+            var pageKeywords = config.page.keywordIds;
+            var hasKeywordsMatch = pageKeywords && intersection(whitelistedKeywordIds, pageKeywords.split(',')).length > 0;
+            var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article') && !config.page.isMinuteArticle; // may render badly on other types
+            return userHasNeverContributed && commercialFeatures.canReasonablyAskForMoney && worksWellWithPageTemplate && hasKeywordsMatch;
+        };
+
+        var contributeUrl = 'https://contribute.theguardian.com/?';
+
+
+        var message = '...we have a small favour to ask. More people are reading the Guardian than ever but far fewer are paying for it. And advertising revenues across the media are falling fast. So you can see why we need to ask for your' +
+            ' help. The Guardian\'s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your ' +
+            'perspective, too.';
```

Comment:
Bit of a red herring, but I am curious about the reason for the string concatenation?